### PR TITLE
Update and rename TimeOut.md to Timeout.md

### PR DESCRIPTION
**Pull request by AnotherButler**
Fix capitalization, make minor grammar changes and change passive to active voice

### CODE REVIEW

File: docs/APIs/tasks/Timeout.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# TimeOut
+# Timeout
```
Comment (iriark01) [inline docs/APIs/tasks/Timeout.md]: This isn't a word, it's the name of the API. Pretty sure it's TimeOut.

File: docs/APIs/tasks/Timeout.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
 
 Use the Timeout interface to set up an interrupt to call a function after a specified delay.
 
-Any number of Timeout objects can be created, allowing multiple outstanding interrupts at the same time.
+You can create any number of Timeout objects, allowing multiple outstanding interrupts at the same time.
```
Comment (iriark01) [inline docs/APIs/tasks/Timeout.md]: Which would imply that this should also be TimeOut. I may be wrong about the name.

Comment (iriark01) [inline docs/APIs/tasks/Timeout.md]: I don't know what I did to make it yellow; didn't mean it, whatever it was.

Comment (AnotherButler) [inline docs/APIs/tasks/Timeout.md]: After much discussion in the office, the engineers say our docs should say "Timeout".

I don't see any yellow.

Comment (iriark01) [inline docs/APIs/tasks/Timeout.md]: Okay, cool

**Pull request merged**